cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -185,10 +185,9 @@ static int _read(ieee802154_dev_t *dev, void *buf, size_t size, ieee802154_rx_in
 
             /* The number of dB above maximum sensitivity detected for the
              * received packet */
-            info->rssi = -CC2538_RSSI_OFFSET + rssi_val + IEEE802154_RADIO_RSSI_OFFSET;
+            info->rssi = (CC2538_RSSI_OFFSET + rssi_val) - IEEE802154_RADIO_RSSI_OFFSET;
```

Comment:
Should it be capped? Although there should be no risk of detecting signals that low.

```suggestion
            int8_t hw_rssi = (CC2538_RSSI_OFFSET + rssi_val) > -174 ? (CC2538_RSSI_OFFSET + rssi_val) : -174;
            info->rssi = hw_rssi - IEEE802154_RADIO_RSSI_OFFSET;
```

Reply:
I'm fine with it, we never know what kind of bugs may appear in the silicon